config/printer-creality-cr6se-2020.cfg

```diff
@@ -0,0 +1,119 @@
+# This file contains pin mappings for the stock 2020 Creality CR6-SE.
+# To use this config, during "make menuconfig" select the
+# STM32F103 with a "28KiB bootloader" and with "Use USB for
+# communication" disabled.
+
+# To enable the stock probe, in "make menuconfig" select
+# "Enable extra low-level configuration options" and set "!PA5"
+# for "GPIO pins to set at micro-controller startup".
```

Comment:
Can you explain why this is needed?  It's preferable to not modify the micro-controller build if possible.  Generally, most run-time pins can be set using a `[static_digital_output xxx]` config section (see config/generic-rambo.cfg for an example).

Reply:
This is needed to correctly initialize the daughterboard that handles the loadcell in the tool head. This has been taken directly from the Marlin implementation. Without it I had unexpected behavior.